proxy/src/coco/source.rs

```diff
@@ -213,7 +213,7 @@ pub enum Revision {
         /// Name of the branch.
         name: String,
         /// The remote peer, if specified.
-        peer_id: Option<peer::PeerId>,
+        peer_id: Option<P>,
```

Comment:
Guess unrelated to the changes at hand, but when can the `peer_id` be `None`?

Reply:
When we're not browsing a peer's Branch, i.e. just local browsing